arch/arm/src/stm32/stm32_sdio.c

```diff
@@ -1016,9 +1021,24 @@ static uint8_t stm32_log2(uint16_t value)
 
 static void stm32_dataconfig(uint32_t timeout, uint32_t dlen, uint32_t dctrl)
 {
-  uint32_t regval = 0;
+  uint32_t clkdiv;
```

Comment:
```suggestion
  uint32_t clkdiv   = 0;
```
nit: since regval is being initialized, `clkdiv` could also be.

Reply:
Thanks. All Vars are initialized in the body, So I opted to remove the wasteful inits.